app/controllers/observations_controller.rb

```diff
@@ -29,4 +29,21 @@ class ObservationsController < ApplicationController
     # an observation from a species_list, but I can't figure out how.
     :edit
   ]
+
+  # Defined here, rather than in a module,
+  # because the superclass needs a class instance variable
+  @dispatch_table_for_index_subactions = {
+    advanced_search: :advanced_search,
+    pattern: :observation_search,
+    look_alikes: :observations_of_look_alikes,
+    related_taxa: :observations_of_related_taxa,
+    name: :observations_of_name,
+    user: :observations_by_user,
+    location: :observations_at_location,
+    where: :observations_at_where,
+    project: :observations_for_project,
+    by: :index_observation,
+    q: :index_observation,
+    id: :index_observation
+  }.freeze
```

Comment:
Question: How best to make this variable available to the superclass?
I'd like to put it in `app/controllers/observations_controller/index.rb`, but it blows up there.